docs/semgrep-ci/configuring-blocking-and-errors-in-ci.md

```diff
@@ -3,59 +3,63 @@ slug: configuring-blocking-and-errors-in-ci
+This article documents how Semgrep handles blocking findings and errors and how you can change Semgrep's default behavior.
 
-## Default configuration of blocking findings and error suppression
+## Blocking findings
 
-Semgrep can be used to block pull requests (PRs) or merge requests (MRs) when it matches a blocking finding. When one or more findings is blocking, Semgrep returns exit code `1`, and you can use this result to set up additional checks to enforce a block in your CI/CD pipeline, such as not allowing merge of the PR/MR. This action applies to both full scans and [diff-aware scans](/semgrep-code/glossary#diff-aware-scan).
+Blocking findings are those identified by Semgrep Code using rules defined in Semgrep AppSec Platform's [Policies page](https://semgrep.dev/orgs/-/policies) and are set to **Block** mode. You can avoid blocking findings by removing rules or by switching the rule mode to **Monitor**, **Comment**, or **Disabled**.
 
-Blocking findings can be defined as:
+If you do **not** use Semgrep AppSec Platform with Semgrep in CI (that is, you are using a **stand-alone setup**), all Semgrep findings are blocking findings. The existence of any findings means that Semgrep returns an exit code of `1`, which you can use to block your PRs or MRs.
 
-- Findings defined in the [Policies page](https://semgrep.dev/orgs/-/policies) of Semgrep AppSec Platform. Avoid blocking findings by removing rules from the **Block** rule mode of the [Policies page](https://semgrep.dev/orgs/-/policies).
-- If you do **not** use Semgrep AppSec Platform with Semgrep in CI (that is, you are using a **stand-alone setup**), blocking findings encompass all Semgrep findings. Any finding in this setup blocks your PRs or MRs.
+## Semgrep's default behavior regarding blocking findings and errors
 
-By default, Semgrep does not block your pipeline when it encounters an internal error. In case of an internal error, Semgrep sends an anonymous crash report to a crash-reporting server and does not block your CI pipeline. To change the default configuration, see the sections below.
+When Semgrep identifies one or more blocking findings, it returns exit code `1`. You can use this result to set up additional checks to enforce a block in your CI/CD pipeline, such as not allowing the merge of the PR/MR. This action applies to both full scans and [diff-aware scans](/semgrep-code/glossary#diff-aware-scan).
 
-:::tip
 The process to enforce a block on a PR or MR after Semgrep exits with error code `1` is dependent on your CI provider. Review your CI provider's documentation for further information.
+
+If Semgrep encounters an internal error, it sends an anonymous crash report to a crash-reporting server and returns exit code `0` to prevent blocking your CI pipeline.
+
+:::info
+See [CLI reference](/cli-reference#exit-codes) for more information about Semgrep exit codes.
 :::
 
 ## Configuration options for blocking findings and errors
```

Comment:
```suggestion
## Configuration options for blocking findings and errors in CI
```
Just to separate this from SMS, where I don't think the user can configure the command.